static/templates/topic_sidebar_actions.hbs

```diff
@@ -34,6 +34,15 @@
         </a>
     </li>
     {{/if}}
+
+    {{#if has_starred}}
+    <li>
+        <a href="#" class="sidebar-popover-unstar-all-in-topic" data-stream-id="{{ stream_id }}" data-topic-name="{{ topic_name }}">
+            <i class="fa fa-star-o" aria-hidden="true"></i>
+            {{#tr this}}Unstar all messages in topic{{/tr}}
```

Comment:
The PR looks good to me in general. I am just concerned that users will get confused by the "all" word here. @timabbott might have something to say about it when he gets time to review this.